### ISSUE TITLE
Regression

### ISSUE THREAD

**Issue by tourniquet555** [Microsoft/CNTK#122]
I'm having trouble getting a simple regression working properly. The .cntk file and training data are below, configured to write out the labels and features when run with command="Basic_Regression_Output". The details of the CN aren't important, just the loading of the "labels".

The resulting output file BasicRegressionOutput.features looks like what I expected (the first two columns of the three column training data, 5000 rows), but the BasicRegressionOutput.labels file (which I expect to be the third column of the training data, 5000 rows) contains just three zeroes.

If someone can point out what I'm doing wrong, I'd greatly appreciate it.

THE .cntk config file:

https://gist.github.com/tourniquet555/224407efb59f33b70d73

The training/testing data:

https://gist.github.com/tourniquet555/bef851d7600d66ab329c

The output features (RIGHT)

https://gist.github.com/tourniquet555/c0a547f27bb85c55e110

The output labels (WRONG?)
https://gist.github.com/tourniquet555/ca5aca4ecca75fe62ea1

**Comment by tourniquet555** [comment]
Has anyone successfully used CNTK for regression yet?

**Comment by 1132520084** [comment]
I also have the same question.
Is there someone can tell me how to set the `labelDim` and `labelMappingFile`. An example would be the best solution .
Thanks a lot.

**Comment by srex212** [comment]
Hi @tourniquet555 

I, too, have been looking to implement a simple regression example and I think I have stumbled across something.  Here are my tweaked files from your examples. Hopefully, this is useful.

[BasicRegression.zip](https://github.com/Microsoft/CNTK/files/261886/BasicRegression.zip)

I think you are outputting the wrong nodes to get the values you want.  I have also found that the network topology is too complex and over-fits on your training data. I have a very simple network topology that seems to give the correct results. For regression, labelDim and labelMappingFile shouldn't be specified.
